.eslintrc.js

```diff
@@ -271,6 +271,7 @@ module.exports = {
         '@typescript-eslint/no-explicit-any': 0,
         '@typescript-eslint/explicit-function-return-type': 0,
         '@typescript-eslint/no-var-requires': 0,
+        '@typescript-eslint/no-this-alias': 0,
```

Comment:
To address linting error at https://github.com/stripe/stripe-node/blob/84d3ec43b0dd582fee811f2330f2001095e90475/src/utils.ts#L226

Reply:
Is it possible to disable this rule next to the line?

Reply:
Yep! I did initially, but I wasn't sure if it would be weird to have the comment `// eslint-disable-next-line @typescript-eslint/no-this-alias` in the compiled JS.